### DISCUSSION TITLE
✅ Auto Preload fonts & images in Elementor (performance)

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21534] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

It would be really great if you added the ability to preload custom fonts. This makes users have a great experience when visiting the website.
Also, if the images at the beginning of each page are automatically preloaded, the performance will be greatly improved. I run these two all the time on my websites and they work great.

These two features can be added experimentally first and later integrated into the core. 

### Describe the solution you'd like

🔥 Auto Preload fonts & images

**Comment by nicholaszein** [comment]
Hello!

These features are already present in Elementor. All font files are prefetched, and images above the fold get high priority when loading while using both the Optimized Image Loading and Lazy Load Background Images.

Best regards
